.github/workflows/main.yml

```
@@ -24,11 +24,14 @@ jobs:
         python -m pip install --upgrade pip
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
-    - name: Test with pytest  ⚙️
+    - name: Test with pytest ⚙️
       run: make test
-    - name: Lint with flake8  ⚙️
+    - name: Lint with flake8 ⚙️
       run: make lint
       if: matrix.python-version == 3.8
+    - name: Check formatting with black ⚙️
+      run: black --check --target-version py38 birdy tests
```

```suggestion
      run: black --check --target-version py36 birdy tests
```
I wouldn't put the conventions up to py38 if we are supporting building on platforms starting at py36. Could potentially introduce new features not supported in python<3.8.